HelpSource/Classes/LFDNoise0.schelp

```diff
@@ -30,33 +30,33 @@ classmethods::
 method::ar, kr
 
 argument::freq
-Approximate rate at which to generate random values.
+Approximate rate at which random values are generated.
 
 argument::mul
-Output will be multiplied by this value.
+Output is multiplied by this value.
 
 argument::add
-This value will be added to the output.
+This value is added to the output.
 
 Examples::
 
-code::
-// try wiggling mouse quickly;
-// LFNoise frequently seems stuck, LFDNoise changes smoothly.
+Code::
+// Wiggle mouse in a fast pace;
```

Comment:
"in a fast pace" is incorrect english (should be "at" not "in") - really there's no reason to change this line